src/components/ContactForm/contactForm.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import '../../assets/main.css';
```

Comment:
You don't need to import this it is already imported in the index.js so it going to apply to the whole app anyway just delete this line